.travis.yml

```diff
@@ -159,6 +159,30 @@ before_install:
       brew install jemalloc;
       brew install gsl;
 
+      PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])')
```

Comment:
It looks like Travis didn't actually do anything with this update, as in I can't find an associated build. It does look like it needs a whole bunch of semicolons, though, which might be the issue.